.github/renovate.json5

```diff
@@ -9,7 +9,10 @@
     // Enable only the regex manager. Go dependencies are managed by Dependabot.
     "regex",
     // Managers for helm and helm-values. Go dependencies are managed by Dependabot.
-    "helmv3"
+    "helmv3",
+    "github-actions",
+    "dockerfile",
+    "gomod"
```

Comment:
I think you could also just completely remove the `enabledManagers` section and it will enable "all" managers.